src/Stripe.net/Infrastructure/Util.cs

```diff
@@ -0,0 +1,111 @@
+namespace Stripe.Infrastructure
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+
+    internal static class Util
```

Comment:
Not really a serious comment, but if the library had avoided a general `Util` class this long, it might be worth avoiding introducing one. Like make this called `StripeTypeConverter` or `StripeTypeInstantiator` or something like that instead.

The reason is essentially that if a `Util` is available, it's a very tempting place for people to throw underdesigned one-offs if they don't want to try and think of a better place to put it.

Reply:
Very good point. Most of our other libraries have a `Util` equivalent, but you're right that it's kind of an antipattern.